### PR TITLE
Docs/move server docs

### LINKED ISSUE #177: Move the server documentation to the main repo
We currently have a small slice of Exoframe docs related to server located in [server repo](https://github.com/exoframejs/exoframe-server). 
This is proving to be pretty inconvenient. We need to move the important bits from that readme into this repo's [docs folder](https://github.com/exoframejs/exoframe/tree/master/docs) and replace the readme stuff with link to new file.

### PR DESCRIPTION
**Pull request by andrasbacsai**
Fixes #177

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,18 +63,6 @@ exoframe
 
 You can find a list of all commands and options in the [docs](./docs/README.md).
 
-## Docs
```
Comment (yamalight) [inline README.md]: The docs have to be removed from server repo, this bit should still be here. 🤔

File: docs/ServerConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+
+## Configuration
```
Comment (yamalight) [inline docs/ServerConfiguration.md]: Rename to "Server Configuration"

File: docs/ServerConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+# path to folder with authorized_keys, default "~/.ssh"
+publicKeysPath: '/path/to/your/public/keys'
+
+# whether Exoframe server whould be running in swarm mode, default "false"
+swarm: false
+```
+
+_Warning:_ Most changes to config are applied immediately. With exception of Letsencrypt config. If you are enabling letsencrypt after Traefik instance has been started, you'll need to remove Traefik and then restart Exoframe server for changes to take effect.
+
+## Docs
```
Comment (yamalight) [inline docs/ServerConfiguration.md]: This should be removed

File: docs/ServerInstallation.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  --label traefik.frontend.rule=Host:exoframe.your-host.com \
+  --label traefik.port=8080 \
+  --constraint=node.role==manager \
+  --name exoframe-server \
+  exoframe/server
+```
+
+Note that both Exoframe server and Traefik will be run on your manager node.
+
+## License
```
Comment (yamalight) [inline docs/ServerInstallation.md]: This should be removed

**Comment by andrasbacsai** [comment]
I hope I did it right. This is my first PR. :)

**Comment by yamalight** [comment]
@andrasbacsai this looks great, thank you! 👍

**Pull request merged**